cmd/amazon-cloudwatch-agent/amazon-cloudwatch-agent.go

```diff
@@ -58,7 +59,7 @@ var fTest = flag.Bool("test", false, "enable test mode: gather metrics, print th
 var fTestWait = flag.Int("test-wait", 0, "wait up to this many seconds for service inputs to complete in test mode")
 var fSchemaTest = flag.Bool("schematest", false, "validate the toml file schema")
 var fConfig = flag.String("config", "", "configuration file to load")
-var fOtelConfig = flag.String("otelconfig", "", "YAML configuration file to run OTel pipeline")
+var fOtelConfig = flag.String("otelconfig", paths.YamlConfigPath, "YAML configuration file to run OTel pipeline")
```

Comment:
This is the primary purpose of the PR. The rest is just refactoring which I am happy to remove/reduce if anyone has suggestions.

Reply:
I know we have a reason to add this default path because customers are having issues, but it's interesting that we'll have a default for the YAML, but not the TOML.

Also TIL that if the TOML config path is empty, then telegraf will try to find the config at a default path https://github.com/aws/telegraf/blob/aws-cwa-release-1-23-0/config/config.go#L771. I don't think customers know about or use those paths, but I guess just in case, we shouldn't set `fConfig` to the default TOML path.

Reply:
Okay wow. I did not know that either. I was considering making the toml config option have a default as well, but now that you said that I am glad I did not.